src/options.c

```diff
@@ -441,7 +442,9 @@ JOHN_USAGE_FORK \
 "--input-encoding=NAME      input encoding (alias for --encoding)\n" \
 "--internal-codepage=NAME   codepage used in rules/masks (see doc/ENCODINGS)\n" \
 "--target-encoding=NAME     output encoding (used by format, see doc/ENCODINGS)\n" \
-"--tune=HOW                 tuning options (auto/report/N)\n"
+"--tune=HOW                 tuning options (auto/report/N)\n" \
+"--force-keys               set up terminal for reading keystrokes even if we're\n" \
+"                           not the foreground process"
```

Comment:
Perhaps we should group our huge list of options options better. Maybe first really important things like --wordlist and --incremental, then lots of random options in alphabetical order (either sorted by option name, or by description), then the rest (optional OpenCL etc) and lastly the --format